tests/PHPRedisFeatureRequesterTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace LaunchDarkly\Tests;
+
+use LaunchDarkly\Integrations\PHPRedis;
+
+/**
+* @requires PHP >= 7.0
+*/
+class PHPRedisFeatureRequesterTest extends FeatureRequesterTestBase
+{
+    const PREFIX = 'test';
+
+    /** @var \Redis */
+    private static $redisClient;
+    
+    public static function setUpBeforeClass()
+    {
+        if (!static::isSkipDatabaseTests()) {
+            $redis = new \Redis();
+            $redis->connect('127.0.0.1', 6379);
+            self::$redisClient = $redis;
+        }
+    }
```

Comment:
Similar to previous comment: even though I know it's possible to set an environment variable to skip all of the database tests, I don't think it's a good idea to have a unit test class that can't possibly work unless a specific _extension_ is installed (since installing things with pecl is not something that is always possible in any given environment). So I think I would prefer to have some logic here that skips these tests if phpredis is not installed. This should work:

```
    public static function setUpBeforeClass()
    {
        if (!static::isSkipDatabaseTests() && extension_loaded('phpredis')) {
            $redis = new \Redis();
            $redis->connect('127.0.0.1', 6379);
            self::$redisClient = $redis;
        }
    }

    public function setUp()
    {
        if (!static::isSkipDatabaseTests() && !extension_loaded('phpredis')) {
            $this->markTestSkipped('phpredis is not installed');
        } else {
            parent::$setup();
        }
    }
```

Reply:
I did use [@require](https://phpunit.readthedocs.io/en/8.4/incomplete-and-skipped-tests.html#skipping-tests-using-requires) to try to limit the test to only PHP 7+, but now that you mention it, checking if the extension is installed if probably a better choice. 
I just pushed the change on the @require, since that would avoid running the test altogether, but I'm open to using your suggestion if you think it's better.

Reply:
I wasn't sure if `@require` was supported in this old version of phpunit, but it looks like it is so that's fine.

Reply:
I just noticed that the CI step you added to install the extension is being run in the 5.6 build as well as the 7.x ones, even though it can't work in 5.6. But I was planning to do some refactoring of the CI script anyway so I can fix that after merging this; it's not actually breaking the build (though I'm not sure why it's not).